tools/pi_db_tools/align_dbspec.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+import sys
+import os
+import argparse
+import re
+from Bio import SeqIO
+
+
+def create_spectra_maps(specfiles, dbfiles, frregex, firstfr):
+    """Output something like
+    {'fr01', 'fr04'} # Normal filename set
+    and
+    {'fr03': ['fr02', 'fr03']}  # pool definition
+    and
+    {'fr04': 'fr04', 'fr04b': 'fr04'}  # rerun fraction, rerun may also be pool
+    """
+    specrange = get_fn_fractionmap(specfiles, frregex)
+    to_pool = []
+    poolmap, rerun_map, normal_fns = {}, [], set()
+    for i in range(0, len(dbfiles)):
```

Comment:
You could use `for i,dbfile in enumerate(dbfiles)`

Reply:
Yeah but my pythong linter will complain because I a don't use the `dbfile` variable then.